src/en/developer-leadership.md

```diff
@@ -167,7 +169,7 @@ primitives. As a charm author, this is unlikely to be the best use of your time
 -- extending the `hacluster` charm to cover your use case is likely to be more
 efficient.
 
-### ...render it likely that a long-lived process runs on one unit at a time?
+^# ...render it likely that a long-lived process runs on one unit at a time?
```

Comment:
Agreed. I also don't like that it is followed by a note box that says:

> Note: this approach is not reliable. It may be good enough for some workloads, but don't use it unless you understand the forces in play and the worst possible consequences for your users...

Wasn't sure how deep into editing existing text I wanted to go when the issue I was addressing just asked for foldouts. However, there is much editing to be done.